app/controllers/application_controller.rb

```diff
@@ -1,43 +1,43 @@
 class ApplicationController < ActionController::Base
-    def log_in(user)
-      session[:user_id] = user.id
+  def log_in(user)
+    session[:user_id] = user.id
 
-      update_cookies(user)
-    end
+    update_cookies(user)
+  end
 
-    def log_out
-      current_user.forget
-      current_user_is(nil)
-      session.delete(:user_id)
-      cookies.delete(:user_id)
-      cookies.delete(:remember_token)
-    end
+  def log_out
+    current_user.forget
+    current_user_is(nil)
+    session.delete(:user_id)
+    cookies.delete(:user_id)
+    cookies.delete(:remember_token)
+  end
 
-    def logged_in?
-      current_user
-    end
+  def logged_in?
+    current_user
+  end
 
-    def current_user_is(user)
-      @current_user = user
-    end
+  def current_user_is(user)
+    @current_user = user
+  end
 
-    def current_user
-      if (user_id = session[:user_id])
-        @current_user ||= User.find_by(id: user_id)
-      elsif (user_id = cookies.signed[:user_id])
-        user = User.find_by(id: user_id)
-        if user && user.authenticated?(cookies[:remember_token])
-          log_in user
-          current_user_is(user)
-        end
+  def current_user
+    if (user_id = session[:user_id])
+      @current_user ||= User.find_by(id: user_id)
+    elsif (user_id = cookies.signed[:user_id])
+      user = User.find_by(id: user_id)
+      if user&.authenticated?(cookies[:remember_token])
+        log_in user
```

Comment:
- [ ] This kind of methods (non-crud specific) are better off in the helpers, you can include them afterward in the controller as you would with any other ruby module.

Reply:
Thanks for the feedback, just fixed it.